### ISSUE TITLE
wrong doc for fast means

### ISSUE THREAD

**Issue by nnorwitz** [opencv/opencv#7899]
http://docs.opencv.org/2.4.10/modules/photo/doc/denoising.html?highlight=denoise#fastnlmeansdenoisingcolored

The parameter is listed as hColor in the signature, but later referenced as hForColorComponents.  Also enought is mispeeled and should be enough.

I didn't check other versions of the doc to see if there was the same problem.

**Comment by rahulchhabra07** [comment]
These issues in  https://github.com/opencv/opencv/blob/master/modules/photo/include/opencv2/photo.hpp are already resolved. The spelling error along with several others is resolved here: #4050.

**Comment by piyushgoel997** [comment]
Hi, I'm new here and wish to work on this issue so can you guide me.

**Comment by gautamchitnis** [comment]
The issue seems to have been resolved in the .hpp file but the website still has same issue. Can anyone tell me what can be done to resolve it?
http://docs.opencv.org/2.4.10/modules/photo/doc/denoising.html?highlight=denoise#fastnlmeansdenoisingcolored

**Comment by alalek** [comment]
2.4.10 documentation is freezed and will not be changed anymore. You should look on active "2.4" maintenance branch instead. Anyway there is no fix on "2.4" yet - "master" branch has this fix only.

To fix it on branch "2.4" you should prepare patch (PR - pull request) via GitHub for "2.4" branch (BTW, "2.4" documentation is stored separately and there are no documentation parts in .hpp files).

**Comment by krishraghuram** [comment]
I cloned the opencv repo and found documentation under "/modules/photo/doc/denoising.rst"
I also modified the rst file to fix the mentioned bug. 
Side Note - The bug only mentions inconsistency in the "fastNlMeansDenoisingColored" documentation, but the same bug also exists in "fastNlMeansDenoisingColoredMulti" documentation. 

Im confused as to how I should build the documentation before submitting a PR. Do I use "sphinx-build"?
